.changeset/polite-knives-build.md

```diff
@@ -0,0 +1,17 @@
+---
+'@backstage/plugin-kubernetes-backend': minor
+---
+
+**BREAKING** You can now select `single` kubernetes cluster that the entity is part-of from all your defined kubernetes clusters, by passing `backstage.io/kubernetes-cluster` annotation with the defined cluster name.
```

Comment:
Note: Just to confirm as I might be misunderstanding things 🙂 .
Is it really breaking? My initial understanding was that the new annotation is optional, and those not using it will continue using the multi-tenant method as it was before.

Reply:
Since there is a new ServiceLocator `singleTenant`, I added breaking changes. Yes, it's an optional and new feature. I will remove the breaking statement.